### ISSUE TITLE
Maintain a list of FOSS-friendly employers

### ISSUE THREAD

**Issue by dboddie** [pybee/paying-the-piper#59]
It would help those of us who want to work on Free and Open Source Software to know which employers hire developers, writers and other roles to work on FOSS projects.

Many of the existing job sites are extremely poor at giving applicants enough information to know what a job requires, let alone give an insight into whether the employees work on FOSS. Partly that's due to a lack of transparency in the recruitment industry and poor tools for searching and filtering results.

Sometimes it's easier to look for promising employers then check if they are hiring than it is to start from the list of jobs available. A list of FOSS-friendly employers would help with that. It wouldn't even require their cooperation - you could post links to pages on employers' sites where they promote FOSS in their working environments.

**Comment by ghost** [comment]
:+1:

I have seen already a project doing so on GitHub, dont remember the name, it was just an .md

**Comment by dboddie** [comment]
A list of employers by region would be useful for those of us not looking to move half way round the world for work. Maybe it would even be possible to get one or two of the larger/friendlier companies to sponsor a page/site like that for increased visibility.

**Comment by ghost** [comment]
Get remote already, geo-limited job dont benefit programmers at all, even more for FOSS.
imho

See:  https://about.gitlab.com/2015/04/08/the-remote-manifesto

**Comment by dboddie** [comment]
Yes, highlighting employers who support remote working would be good, too.

**Comment by ghost** [comment]
This gal here done an awesome list, maybe it just need an extra column with **works with FOSS ?**

https://github.com/jessicard/remote-jobs#remote-friendly-companies

:smile_cat:

**Comment by dboddie** [comment]
If only it was as simple as joining those tables. :smile: Perhaps opening an issue there would be good, or maybe forking the repository and submitting a pull request later.
